features/step_definitions/common.ts

```diff
@@ -1,6 +1,28 @@
-import { When } from "@cucumber/cucumber";
+import * as assert from "assert";
+import { Then, When } from "@cucumber/cucumber";
 import { execCliKintoneSync } from "../ultils/helper";
 
 When("I run the command with args {string}", function (args: string) {
   this.response = execCliKintoneSync(args);
 });
+
+Then("I should get the exit code is non-zero", function () {
+  assert.notEqual(this.response.status, 0);
+});
+
+Then("I should get the exit code is zero", function () {
+  assert.equal(this.response.status, 0);
+});
+
+Then(
+  "The output error message should contain {string}",
```

Comment:
Same as the "The output message should contain {string}".
It's better to rename the step to explain its implementation.